{
  handleException(exception,"FATAL");
  myTrouble[0]=true;
}
